modules/schat.cpp

```diff
@@ -226,9 +230,15 @@ class CSChat : public CModule
 					Table.SetCell("Host", pSock->GetRemoteIP());
 					Table.SetCell("Port", CString(pSock->GetRemotePort()));
 					SSL_SESSION *pSession = pSock->GetSSLSession();
+#ifdef HAVE_OPAQUE_SSL_SESSION
+					if (pSession)
+						Table.SetCell("Cipher", SSL_CIPHER_get_name(SSL_SESSION_get0_cipher(pSession)));
+#else
 					if (pSession && pSession->cipher && pSession->cipher->name)
 						Table.SetCell("Cipher", pSession->cipher->name);
-
+#endif
+					else
+						Table.SetCell("Cipher", "None");
```

Comment:
Alternatively, you could also do `#define SSL_CIPHER_get_name(session) ((session)->cipher) ? (session)->cipher->name : "None")` instead of a static inline method.